meta/releasing.md

```diff
@@ -4,6 +4,51 @@ The whole specification is now released as a single unit/artifact. This document
 the process for releasing the specification and a description of how the (public)
 machinery works.
 
+## Timeline
+
+The spec is released each calendar quarter. The target release dates are within the
+following ranges:
+
+* Q1: January 20-27 (critically, before FOSDEM).
+* Q2: May 20-27.
+* Q3: August 20-27.
+* Q4: November 1-15 (before recurring November conflicts).
```

Comment:
.... (what happens in November?)

Reply:
IETF meetup, people start disappearing for holidays, most projects are wrapping up, etc.

Reply:
have stuck a mention in the doc so we don't forget